scripts/process_schemas.py

```diff
@@ -106,8 +106,8 @@ def main():
     parser.add_argument(
         "version", help="Version number of the schema we're compiling")
     parser.add_argument(
-        "--schema-path", default="../schemas",
-        help="Path to schemas. Default is ../schemas")
+        "--schema-path",
+        help="Path to schemas.", required=True)
```

Comment:
Required 'options' make for a weird UI --- can we make this a positional argument instead please?